CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contributing to this project
+
+You can contribute to this project by creating a [Pull Request](https://github.com/Ldoppea/first-steps-dev/pulls). Contributors must abide to this project [Code of Conduct](CODE_OF_CONDUCT.md)
+
+For example, you can propose a new link, new category, delete a linked that became invalid, fix typos or tags, or propose new ideas.
+
+Be sure to have a look to this checklist before submitting a new PR:
+
+- [ ] The proposed link should target content preferably in the same language as the list it's added to. You can however propose English content to non-english lists if it's high quality and add the `[🇺🇸]` suffix after the link.
+- [ ] The linked content is free and non-gated (i.e. it should not require a subscription to access it).
+- [ ] The linked content does not contain any offensive material of any sort that would result in a breach of the [Code of Conduct](CODE_OF_CONDUCT.md).
```

Comment:
Maybe we can add something like:
- The linked content is added to an existing category or a new created one. If you are not sure of the category you may create your PR as a draft and start a discussion about it.